biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -2295,13 +2436,19 @@ subroutine fuse_2_patches(csite, dp, rp)
     inv_sum_area = 1.0_r8/(dp%area + rp%area)
     
     rp%age = (dp%age * dp%area + rp%age * rp%area) * inv_sum_area
+    rp%age_since_anthro_disturbance = (dp%age_since_anthro_disturbance * dp%area &
+         + rp%age_since_anthro_disturbance * rp%area) * inv_sum_area
```

Comment:
what is this in the case of primary patches?

Reply:
good question. I guess that it is only used or incremented for secondary patches, but maybe it needs a special fill value for primary patches?

Reply:
looking at this (here: https://github.com/NGEET/fates/blob/master/biogeochem/EDPatchDynamicsMod.F90#L1884), it seems that the current code sets this as -1 for primary patches.  there's a note to replace it with fates_unset_r8 when possible.  I guess I'll do that for consistency?